scripts/base/frameworks/cluster/nodes/logger.zeek

```diff
@@ -23,4 +23,7 @@ redef Log::default_rotation_interval = 1 hrs;
 redef Log::default_mail_alarms_interval = 24 hrs;
 
 ## Use the cluster's archive logging script.
+
+@if ( ! Supervisor::is_supervised() )
```

Comment:
Since most likely no one will have the `archive-log` script in their `PATH`, file rotation will emit errors that it can't be found.  I'm thinking that `redef` doesn't quite belong there in the first place -- if it's a contrivance for ZeekControl to have set up `PATH`, it could have instead generated its own script with that `redef` (could even generate it with absolute path if it wanted).